README.md

```diff
@@ -67,6 +71,20 @@ Deploy:
 kubectl kustomize config/default | kubectl apply -f -
 ```
 
+At this point you should see the `controlplane` and `dataplane` `Pods` running
```

Comment:
```suggestion
At this point you should see the `controlplane` and `dataplane` pods running
```

Reply:
Well, in this case I'm trying to actually indicate that we're talking about Kubernetes `Pods` which is why it's highlighted this way, and CamelCased.

Reply:
updated in a00556c430e45bd4b84efd6f05a0e4fb3ea99c32